commcare-cloud/commcare_cloud/getinventory.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```

Comment:
I guess it's still worth keeping this file executable (as this line does)? I occasionally run `./scripts/inventory/getinventory.py` directly to print out the IP/DNS name of a machine, rather than opening the inventory file up in my editor and searching, so now I guess I'd run `commcare-cloud/commcare_cloud/getinventory.py`. Might be worth recasting this as `commcare-cloud <env> lookup <server>` but happy to leave that for another day.

Reply:
Ha, I completely overlooked this line. It doesn't work that way anymore (I removed that functionality). Unfortunately my changes are not easy to see because I changed too much and git didn't realize the file was moved. If you want I will rebase this and do the move and refactoring in separate commits to make it more obvious.

I'll be happy to add the `lookup` command in a separate PR.